modules/edge-treasury-reward/src/lib.rs

```diff
@@ -50,6 +50,20 @@ decl_module! {
 				);
 			}
 		}
+
+		/// Sets the fixed treasury payout per minting interval.
+		#[weight = 5_000_000]
```

Comment:
is there a reason you chose 5,000,000?

Reply:
Yes: keeping the pricing consistent with similar substrate accessors.